tensorflow/core/kernels/decode_raw_op.cc

```diff
@@ -73,6 +73,13 @@ class DecodeRawOp : public OpKernel {
     for (int64 i = 0; i < flat_in.size(); ++i) {
       const T* in_data = reinterpret_cast<const T*>(flat_in(i).data());
       memcpy(out_data, in_data, str_size);
+      if (port::kLittleEndian != little_endian_) {
```

Comment:
Instead if memcpy + byteswap, just use a custom copy loop for the big endian case, so you don't have to touch the data twice.